src/mlnet/Program.cs

```diff
@@ -20,24 +21,33 @@ namespace Microsoft.ML.CLI
     class Program
     {
         private static Logger logger = LogManager.GetCurrentClassLogger();
+
         public static void Main(string[] args)
         {
-            var telemetry = new MlTelemetry();
+            Telemetry.Telemetry.Initialize();
             int exitCode = 1;
+            Exception ex = null;
+            var stopwatch = Stopwatch.StartNew();
+
+            var mlNetCommandEvent = new MLNetCommandEvent();
+
             // Create handler outside so that commandline and the handler is decoupled and testable.
             var handler = CommandHandler.Create<NewCommandSettings>(
                 (options) =>
              {
                  try
                  {
+                     // Send telemetry event for command issued
+                     mlNetCommandEvent.AutoTrainCommandSettings = options;
```

Comment:
Not grok'ing the design here.  If it's a generic ml.net command event, then it shouldn't have anything automl specific.  If it's automl.net specific thing then why not call it that.  I'm trying to prevent an uber (garbage can) class where every command tosses whatever it wants.    The initial MlTelemetry class was flawed in that way too since I didn't anticipate having more than one event per command, but at least it didn't have any command- or event-specific members, just methods plus global stuff like logging command line params.

Reply:
+1. If I were designing this from scratch, I would not do this.
The original telemetry event was called "mlnet-command". It has AutoML-specific properties like hash of the train file name and name of the learning task (ie multiclass classification).
I would like to change the name of this event from "mlnet-command" to "auto-train-command", but if we do that, then we'd bifurcate our prod metrics & events. For backwards compatibility, I went this route -- but perhaps we should just make this breaking change. I'm not sure